sklearn/ensemble/_forest.py

```diff
@@ -1193,9 +1215,10 @@ class RandomForestClassifier(ForestClassifier):
         Whether bootstrap samples are used when building trees. If False, the
         whole dataset is used to build each tree.
 
-    oob_score : bool, default=False
+    oob_score : bool or callable, default=False
         Whether to use out-of-bag samples to estimate the generalization score.
-        Only available if bootstrap=True.
+        By default the accuracy score is used. Provide a callable to use a custom
+        scoring function. Only available if bootstrap=True.
```

Comment:
I'm concerned over the term "scoring function". In other parts of the codebase, "scorer" is used to mean a callable with signature "(estimator, X, y)", while in this case it is: `(y_true, y_pred)`. I think we can reduce some confusion by liking to `accuracy_score` directly and use the word "metric".

```suggestion
        By default, :func:`~sklearn.metrics.accuracy_score` is used.
        Provide a callable with signature `metric(y_true, y_pred)` to use a
        custom metric. Only available if `bootstrap=True`.
```

Reply:
I like it. Done